### DISCUSSION TITLE
Textarea min-height

### DISCUSSION THREAD

**Discussion by gyasi-eco** [twilio-labs/paste#3227] [Q&A] [answered]
Hello, I'm currently working on the dev phone and we're using the Textarea component to send messages. 
I was curious if there is a way to change the min-height without using custom css?

**Comment by loreina** [comment]
Hi @gyasi-eco,

We have a [Chat Composer](https://paste.twilio.design/components/chat-composer) component from our Conversations UI Kit that's used for composing rich text messages. Would this be what you're looking for?

For our team to assist best, please share a screenshot and more context into the product so we can understand what you're trying to use Textarea for. Thanks!

**Reply by gyasi-eco** [reply]
Hi @loreina 

We're using the Textarea component for outbound sms for [https://github.com/twilio-labs/dev-phone](url) and specifically to resolve this issue [https://github.com/twilio-labs/dev-phone/issues/167](url). Would Chat Composer be sufficient to replace Textarea as our input component even though the input is used for outbound SMS which doesn't support most rich text.
![Screen Shot 2023-05-11 at 3 59 32 PM](https://github.com/twilio-labs/paste/assets/110051661/1246c6e3-bb4a-4ec9-aa85-02fde9b8bca0)

**Reply by loreina** [reply, marked as answer]
Hi @gyasi-eco, you can use Textarea here but you'll need to handle the value to ensure that line breaks aren't removed.
